remote-config.jsonc

```
@@ -194,47 +182,47 @@
         {
           "message": "A key goal for DDEV in 2024 is to make it possible for maintainer Stas Zhuk to work full-time on DDEV, see https://ddev.com/blog/lets-fund-stas-maintainer/"
         },
```

```suggestion
        {
          "message": "A key goal for DDEV in 2025 is to make it possible for maintainer Stas Zhuk to work full-time on DDEV, see https://ddev.com/blog/lets-fund-stas-maintainer/"
        },
```

Bump year to 2025, assuming this is still a goal.